src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 103
103:                 $result1.Reply.Options.DontFragment | Should -BeTrue
```

Reply:
@SteveL-MSFT I have confirmed with a manual test that this regression comes from .NET Core itself -- a manual use of the Ping.Send() or SendAsync() exhibits this behaviour where a DontFragment setting is ignored (or at least not returned as set from the ping reply).

This appears to be a new regression since the 3.0 release.

This API is... getting _more_ fragile. 😕

Marking this test as pending until this gets fixed.

Reply:
Did someone file an issue?

Reply:
I added a note on the existing tracking issue in coreFX for the Ping APIs.

One of the coreFX folx asked if I could set breakpoints and check what's going on. I may give it a stab, but I'm not really sure what I'm doing there tbh. CoreFX is a very large project, and I'm not familiar with how to work with it in a debugging context, or even build it at all.

Reply:
Should I create an additional tracking issue here as well? We already have a tracking issue for the APIs used by this cmdlet in general I think I can add a note to.

Reply:
I'm good as long as it's tracked somehow.